cli/pkg/config_manager/config_manager.go

```diff
@@ -0,0 +1,489 @@
+		serviceDefs[service.Name] =
+			map[string]any{
+				"name":                  service.Name,
+				"port":                  service.Port,
+				"health_check_path":     service.Uri,
+				"service_type":          service.ServiceType,
+				"path":                  fmt.Sprintf("%s/*", service.Uri),
+				"priority":              service.Priority,
+				"success_codes":         "200-499",
+				"platform_architecture": "arm64",
+				"build": map[string]any{
+					"context":    service.Context,
+					"dockerfile": service.DockerfilePath,
+				},
+			}
+		serviceNames = append(serviceNames, service.Name)
+	}
+	happyConfig.GetData().StackDefaults = stackDefaults
+	happyConfig.GetData().Services = serviceNames
+
+	err = os.MkdirAll(filepath.Dir(bootstrapConfig.HappyConfigPath), 0777)
+	if err != nil {
+		return nil, errors.Wrapf(err, "unable to create folder %s", filepath.Dir(bootstrapConfig.HappyConfigPath))
+	}
+	err = happyConfig.Save()
+
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to save happy config")
+	}
+
+	happyConfig, err = config.NewHappyConfig(bootstrapConfig)
+
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to load happy config")
+	}
+
+	return happyConfig, nil
+}
+
+func ListClusterIds(ctx context.Context, profile, region string) ([]string, error) {
+	b, err := backend.NewAWSBackend(ctx, config.EnvironmentContext{
+		AWSProfile:     util.String(profile),
+		AWSRegion:      util.String(region),
+		TaskLaunchType: util.LaunchTypeNull,
+	})
+	if err != nil {
+		return []string{}, errors.Wrap(err, "unable to create an aws backend")
+	}
+	return b.ListEKSClusterIds(ctx)
+}
+
+func ListHappyNamespaces(ctx context.Context, profile, region, clusterId string) ([]string, error) {
+	b, err := backend.NewAWSBackend(ctx, config.EnvironmentContext{
+		AWSProfile:     util.String(profile),
+		AWSRegion:      util.String(region),
+		TaskLaunchType: util.LaunchTypeK8S,
+		K8S: k8s.K8SConfig{
+			AuthMethod: "eks",
+			ClusterID:  clusterId,
+		},
+	}, backend.WithIntegrationSecret(&config.IntegrationSecret{})) // This will prevent the backend from trying to load the integration secret, as we have not selected the namespace yet
+	if err != nil {
+		return []string{}, errors.Wrap(err, "unable to create an aws backend")
+	}
+	return b.ListHappyNamespaces(ctx)
+}
+
+func findAllDockerfiles(path string) ([]string, error) {
+	logrus.Debugf("Searching for Dockerfiles in %s", path)
+	paths := []string{}
+
+	err := filepath.WalkDir(path, func(path string, d fs.DirEntry, err error) error {
+		if err != nil {
+			return err
+		}
+		if d.IsDir() {
+			if d.Name() == ".terraform" || d.Name() == ".git" {
+				return filepath.SkipDir
+			}
+			return nil
+		}
+		if strings.HasSuffix(path, ".dockerignore") {
+			return nil
+		}
+		if !strings.HasSuffix(path, "Dockerfile") && !strings.Contains(path, "Dockerfile.") {
+			return nil
+		}
+		paths = append(paths, path)
+		return nil
+	})
+
+	return paths, err
+}
+
+func Port(val interface{}) error {
```

Comment:
The name of this function is not clear. It's only validating the interface{} is a valid port, right? maybe isValidPort? or isPort?